common4j/src/main/com/microsoft/identity/common/java/nativeauth/providers/NativeAuthOAuth2Strategy.kt

```diff
@@ -86,23 +85,12 @@ class NativeAuthOAuth2Strategy(
         return config.authorityUrl.toString()
     }
 
-    /**
-     * Makes the initial call to the /signup/start.
-     * @param commandParameters: Attributes provided by the user
-     */
-    fun performSignUpStart(
-        commandParameters: SignUpStartCommandParameters
-    ): SignUpStartApiResult {
-        LogSession.logMethodCall(TAG, "${TAG}.performSignUpStart")
-        return signUpInteractor.performSignUpStart(commandParameters)
-    }
-
     /**
      * Makes the initial call to the /signup/start when the parameters includes password.
      * @param commandParameters: Attributes provided by the user
      */
     fun performSignUpStartUsingPassword(
```

Comment:
Let's rename this to remove the reference to password? And update the javadoc

Reply:
Done